src/ws_tray.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+/*
+* WSTray module handles toggling the state/image of our tray icon.
+*/
+const { app, Tray } = require('electron');
+const path = require('path');
+const is = require('electron-is');
+const osxPrefs = require('electron-osx-appearance');
+
+// private
+// On mac ending the filename in 'Template' tells the os the image is only B+W,
+// this means the os will handle toggling the color for darkMode. Since we need
+// to show notifications we have to handle the toggle of the notification images
+// ourselves.
+const macIcon = path.join(__dirname, '../assets/images/iconLightTemplate.png');
+const macIconLightNotify = path.join(__dirname, '../assets/images/iconLightNotify.png');
+const macIconDarkNotify = path.join(__dirname, '../assets/images/iconDarkNotify.png');
+// On Windows we need larger images so they display correctly. We only need light
+// images on windows since the task bar is dark.
```

Comment:
Do we have anything with transparent backgrounds so that we don't have to worry about colors? Particularly on windows, I think the taskbar can be themed? (Or maybe that was a long time ago...)

Reply:
The icons have alpha set but the foreground is either white or black. I figure we can look deeper if it becomes an issue on windows. On mac the normal behavior is to use template images and then the os handles it... But those can't have color. It would be nice if the display of the notification didn't require swapping the whole image.